RELEASE.md

```diff
@@ -449,9 +451,9 @@ To publish the release:
   * Copy the release notes text from the template at [`docs/templates/release-notes`](https://github.com/freedomofpress/dangerzone/tree/main/docs/templates/)
   * You can use `./dev_scripts/upload-asset.py`, if you want to upload an asset
     using an access token.
-- [ ] Upload the `container.tar.gz` i686 image that was created in the previous step
+- [ ] Upload the `container.i686.tar.gz` and `container.arm64.tar.gz` images that were created in the previous step
```

Comment:
Hm, I think we have kinda started naming our artifacts as `<artifact>-<version>-<arch>.<ext>`. So, would `container-0.7.1-arm64.tar.gz` make more sense here? Note that whatever we choose, we also need to update our `sign-assets.py` script.

Reply:
That's better, thanks. Updated.